core/commands.c

```diff
@@ -126,28 +137,32 @@ static int ventCommand(size_t argc, char *argv[], size_t outbufc, char outbuf[])
   }
 }
 
-static int statusCommand(size_t argc, char *argv[], size_t outbufc, char outbuf[]) {
+static int statusCommand(size_t argc, char *argv[], size_t outbufc,
+                         char outbuf[]) {
   pod_t *pod = get_pod();
   return status_dump(pod, outbuf, outbufc);
 }
 
-static int fillCommand(size_t argc, char *argv[], size_t outbufc, char outbuf[]) {
+static int fillCommand(size_t argc, char *argv[], size_t outbufc,
+                       char outbuf[]) {
   if (start_hp_fill()) {
     return snprintf(outbuf, outbufc, "Entered HP Fill State");
   } else {
     return snprintf(outbuf, outbufc, "HP Fill Pre-Check Failure");
   }
 }
 
-static int standbyCommand(size_t argc, char *argv[], size_t outbufc, char outbuf[]) {
+static int standbyCommand(size_t argc, char *argv[], size_t outbufc,
+                          char outbuf[]) {
```

Comment:
Interesting it seems that I've been guilty of some style violations... I'll have to check my formatting config